src/main/java/seedu/address/logic/commands/help/HelpDescription.java

```diff
@@ -0,0 +1,163 @@
+            + "\n 2. Deleting an Interview: \n"
+            + "Format: 'del -i CANDIDATE_INDEX ROLE' \n"
+            + "Some notice: \n"
+            + "- Existing interview with the matching CANDIDATE_INDEX and ROLE is deleted. \n"
+            + "- Role provided is case-insensitive. \n"
+            + "Examples: \n"
+            + "del -i 1 Senior Frontend Software Engineer \n \n"
+            + "3. Deleting positions: \n"
+            + "Format: 'del -p ROLE' \n"
+            + "Some notice: \n"
+            + "- Existing position with the specified name is deleted. \n"
+            + "- The position name has to match with the position that is to be deleted. \n"
+            + "- Position name provided is case-insensitive. \n"
+            + "Examples: \n"
+            + "del -p Senior Frontend Software Engineer";
+
+    public static final String LIST_COMMAND_DESCRIPTION =
+            "1. List all applicants: \n"
+            + "Format: 'list -a' \n"
+            + "\n 2. Listing all Interview for a candidate: \n"
+            + "Format: 'list -i CANDIDATE_INDEX' \n"
+            + "Examples: \n"
+            + "list -i 1 \n \n"
+            + "3. Listing all positions: \n"
+            + "Format: 'list -p' \n"
+            + "Some notice: \n"
+            + "- Existing position with the specified name is deleted. \n"
+            + "- The position name has to match with the position that is to be deleted. \n"
+            + "- Position name provided is case-insensitive. \n"
+            + "Examples: \n"
+            + "del -p Senior Frontend Software Engineer";
+
+    public static final String FILTER_COMMAND_DESCRIPTION =
+        "View different applicants, interviews and positions in HireLah through various filters. "
+        + "It alters the current display of HireLah and changes the index of the relevant data. \n"
+        + "Format: 'filter DATA_TYPE b/FILTER_TYPE [ARGUMENT]' \n"
+        + "Notice: \n"
+        + "Different data has different filters available, thus requiring different arguments, as such"
+        + "- Applicant (DATA_TYPE: appl): \n"
+        + " + FILTER_TYPE: name, ARGUMENT: n/KEYWORD: View applicants whose name contains the keyword \n"
+        + " + FILTER_TYPE: tag, ARGUMENT: t/TAG1, [t/TAG2, …]: View applicants who have all the tags specified \n"
+        + "- Interview (DATA_TYPE: intvw): \n"
+        + " + FILTER_TYPE: appl, ARGUMENT: n/NAME: View interviews for applicants whose name is specified \n"
+        + " + FILTER_TYPE: date, ARGUMENT: d/DATE: View interviews happening on the specified date "
+        + "(Date provided must be in format YYYY-MM-DD) \n"
+        + "- Position (DATA_TYPE: pos): \n"
+        + " + FILTER_TYPE: name, ARGUMENT: n/KEYWORD: View positions which has "
+        + "the specified keyword in the position name"
+        + "Examples: \n"
+        + "filter appl tag t/school t/friend \n"
+        + "filter intvw date d/2022-03-20";
+
```

Comment:
Maybe we could comment this out first, since we are not able to `filter` in this iteration v1.2?